scripts/precompile.py

```diff
@@ -14,7 +14,7 @@
 
 from reflex.utils import path_ops
 
-from .constants import ELEMENT_TO_PROPS, ELEMENTS
+from ..reflex.components.el.constants import ELEMENT_TO_PROPS, ELEMENTS
```

Comment:
We shouldn't need a relative import, does the normal `from reflex.components.el.constants` work here?